host_init.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# This script is meant to be ran with sudo privileges as a non-root user.
+# Requires the pull-secret file to be present in the current directory.
+# The script will:
+# 1. Configure passwordless sudo for the user
+# 2. Invoke subscription-manager in order to register and activate the subscription (in case of RHEL)
+# 3. Install new packages (like Git, text editor, shell and so on)
+# 4. Clone the dev-scripts repository
```

Comment:
Great question, that's one of the issues I wanted to ask about.
I thought about simply adding a `curl | sh` command to README and removing the `git clone` part from the script.

Reply:
As this is a "preparation" script, I'd like to spare the user from as many actions as possible.
Thus a `curl | sh` seems like the cleanest idea here. Otherwise the user will need to `yum update && yum install git && git clone` and then run the script.